documentation/src/main/asciidoc/user_guide/transactions.adoc

```diff
@@ -244,11 +244,11 @@ Pessimistic transactions are more costly by their nature: each write operation p
 
 Write skews occur when two transactions independently and simultaneously read and write to the same key. The result of a write skew is that both transactions successfully commit updates to the same key but with different values.
 
-The `write-skew` attribute automatically detects and rolls back one of the transactions. However, write skew checks apply only to `REPEATABLE_READ` isolation levels in optimistic transactions.
+Write skew checking is disabled except for {brandname} Library Mode, which defaults to the `REPEATABLE_READ` isolation level with optimistic locking transactions.
```

Comment:
As mentionend in the ISPN-9885 description, the attribute write-skew is no longer available within the configuration. I suppose for both modes embedded/server.
So mention it here is confusing. We should mention that there WAS and attribute like this in legacy releases but now it is gone and enabled/disabled automatically if needed.
@pruivo can give more background

Reply:
@wfink write-skew is mentioned in the config schema. https://docs.jboss.org/infinispan/9.4/configdocs/infinispan-config-9.4.html

Reply:
@wfink also in 10.0. https://docs.jboss.org/infinispan/10.0/configdocs/infinispan-config-10.0.html

Reply:
@wfink I've discussed with @pruivo and have updated. thanks.

Reply:
@wfink I thought write skew was enabled in server mode as well, if the cache is transactional and optimistic?